views/partials/navbar.ejs

```diff
@@ -0,0 +1,129 @@
+<link rel="shortcut icon" href="/assets/favicon.ico">
+<link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
+<link rel="stylesheet" href="/css/nav-style.css">
+
+</head>
+
```

Comment:
Maybe I'm wrong but wouldn't it be better to just edit the Header.ejs with the links and fill the navbar with just the code from the navbar itself?